os/board/common/crashdump.c

```diff
@@ -201,7 +249,7 @@ void board_crashdump(uint32_t cur_sp, void *tcb, uint8_t *filename, int lineno)
 	 * watchdog reset might stall the system, causing crashdump hang.
 	 * Hence let's disable watchdog reset during crashdump process.
 	 */
-	up_watchdog_disable();
+//	up_watchdog_disable();
```

Comment:
Could you let me know the reason of commenting out?

Reply:
This API is defined under the config CONFIG_WATCHDOG_FOR_IRQ (default n when CONFIG_WATCHDOG is enabled) in file  "arch/arm/src/imxrt/**imxrt_wdog**.c", whereas this API in crashdump file is using CONFIG_WATCHDOG conditional compilation only.

Should I add CONFIG_WATCHDOG_FOR_IRQ check as well for it ??

Reply:
@jeongchanKim what do you think about above dependency ??

Reply:
@sunghan-chang @sijupunnnoose Please check this dependency